apps/frontend/src/app/5_pages/RewardsPage/components/TotalRewardsEarned/TotalRewardsEarned.tsx

```diff
@@ -32,6 +32,7 @@ export const TotalRewardsEarned: FC = () => {
           precision={BTC_RENDER_PRECISION}
           dataAttribute="total-earned-rewards"
           isAnimated
+          tooltipClassName="break-all sm:break-normal sm:max-w-full ml-4 sm:ml-0"
```

Comment:
Is there any reason why you didn't implement it at the component level (so it's fixed everywhere)?

Reply:
@tiltom The reason is that I considered this to be an isolated case, because before this did not happen, and all words and long numbers were placed normally, but taking into account your suggestion, we can be insured, and makes sense to implement it at the component level to exclude such problems in the future.

Reply:
Thanks :) Just make sure to test multiple places once it's replaced.